addOns/scripts/src/main/resources/org/zaproxy/zap/extension/scripts/resources/Messages.properties

```diff
@@ -1,3 +1,20 @@
+scripts.automation.desc = Scripts Automation
+scripts.automation.dialog.summary.noAction = No Action specified for script job
+scripts.automation.dialog.summary.run = Run Script: {0}
+scripts.automation.dialog.action = Action:
+scripts.automation.dialog.name = Name:
+scripts.automation.dialog.scriptName = Script Name:
+scripts.automation.dialog.scriptType = Script Type:
+scripts.automation.dialog.title = Script Job
+scripts.automation.error.actionNull = Action is required, but not specified. Following actions are valid: {0}
+scripts.automation.error.actionNotDefined = Specified action '{0}' not defined. Only following actions are valid: {1}
+scripts.automation.error.nofile = Cannot access file:  {0}
+scripts.automation.error.scriptError = Error in Script {0}\\n{1}
```

Comment:
Was the two slashes on purpose?

Reply:
no, thanks. Good catch.

Because it is only the message, I removed the new line entirely